packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -149,6 +151,30 @@ function compressInterval(interval: ISerializedInterval): CompressedSerializedIn
 	];
 }
 
+function startReferenceSlidingPreference(stickiness?: IntervalStickiness): SlidingPreference {
```

Comment:
not sure how well this actually pans out, but maybe it's worth making stickiness required everywhere except public APIs and removing the default handling here?

I'm not sure this is actually more ergonomic, so if you think it isn't feel free to leave the same.

Reply:
I didn't really see places besides this where a non-null stickiness wouldn't cause a breaking change -- but I did update this to just use a default value for stickiness wherever possible, which does simplify some conditions